stellargraph/layer/gcn.py

```diff
@@ -311,8 +311,16 @@ def __init__(
                 num_of_nodes=self.generator.Aadj.shape[0]
             )
 
+        # Activation function for each layer
+        if activations is None:
+            activations = ["relu"] * (n_layers - 1) + ["linear"]
```

Comment:
Having a final activation of linear is appropriate GraphSAGE as, by default, it has L2 normalization applied afterwards, it is perhaps not appropriate for GCN.  I think having all layers default to "relu" would be better.